### DISCUSSION TITLE
Contiki container does not run as non-admin user

### DISCUSSION THREAD

**Discussion by rahav** [contiki-ng/contiki-ng#2559] [Q&A]
After installing the Contiki container on Ubuntu 22.04 LTS, running the Contiki container shows the proverbial

    To run a command as administrator (user "root"), use "sudo <command>".
    See "man sudo_root" for details.

The user invoking the `contiker` command is a member of the `docker` group

    groups
    docker adm cdrom sudo dip plugdev lpadmin lxd sambashare rahav

Running other containers, for example Docker's hello world: `docker run hello-world` or `docker run -it ubuntu bash` do not display above  message.

Running `sudo contiker` will be responded with

    sudo: contiker: command not found

**Comment by pjonsson** [comment]
Isn't that message from inside the container? Here's what it looks like for me:

```
pj@Ubuntu22-VB:~/src/contiki-ng$ contiker
To run a command as administrator (user "root"), use "sudo <command>".
See "man sudo_root" for details.

user@ba448a24c004:~/contiki-ng$ 
```

**Comment by rahav** [comment]
I found the problem. It is not coming from Docker but from Ubuntu. It is a hint that comes from `/etc/bash.bashrc` telling the user they can run commands as admin. When `sudo` is run for the first time, it will create the file `~/.sudo_as_admin_successful` and the hint will not be displayed any more. This would work for the host Ubuntu OS but not for the container. Because the container starts from a Docker image that does not have that `~/.sudo_as_admin_successful` file, the hint will be displayed every time.

With that, I think it is harmless and I am going to close this ticket. Thanks for your help, it made me think...
